languages/python3/template/algorithmia.conf

```diff
@@ -0,0 +1,4 @@
+{
+    "username": "__USER__",
+    "algoname": "__ALGO__"
```

Comment:
nit: would be good to have `language` in here still.  we haven't created it on the backend but probably `python3.7` would be our name

Reply:
👍